log-insights/setup/aiven/04_configure_service_integration.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Step 4: Configure Service Integration'
+install_track_title: 'Log Insights: Setup (Aiven)'
+backlink_href: /docs/log-insights/setup
+backlink_title: 'Log Insights: Setup'
+---
+
+import imgAddSyslogIntegration from './add-syslog-integration.png'
+import imgManageIntegrations from './manage-integrations.png'
+import imgUseSyslogIntegration from './use-syslog-integration.png'
+import imgSyslogIntegrationActive from './syslog-integration-active.png'
+import imgLogInsightsScreenshot from '../log_insights_screenshot.png'
+
+export const ImgAddSyslogIntegration = () => <div style={{display: 'flex', justifyContent: 'center'}}><img alt="Add new syslog service integration in Aiven Console" src={imgAddSyslogIntegration} style={{maxWidth: 500}} /></div>
+
+export const ImgManageIntegrations = () => <img alt="Manage Integrations item in database details page" src={imgManageIntegrations} />
+
+export const ImgUseSyslogIntegration = () => <img alt="Select the Rsyslog integration in the integration list" src={imgUseSyslogIntegration} />
+
+export const ImgSyslogIntegrationActive = () => <img alt="Select the Rsyslog integration" src={imgSyslogIntegrationActive} />
+
+export const ImgLogInsightsScreenshot = () => <img src={imgLogInsightsScreenshot} alt='Screenshot of pganalyze Log Insights feature' />
+
+The last step is setting up the Aiven Service Integration. Go to the
+Service Integration page in your Aiven console, select Syslog from the
+list of options, and click "Add new endpoint".
+
+Fill in the Endpoint form with the internal IP of your collector instance and
+the port you selected in the previous step. Disable TLS (the collector does not
+support receiving logs over TLS, and since you're sending logs over the peered
+VPC connection, the traffic is already private). Leave the format as rfc5424,
+and the other fields blank:
+
+<ImgAddSyslogIntegration />
+
+Once this is done, return to the service details page for your database, and
+click Manage Integrations:
+
+<ImgManageIntegrations />
+
+Scroll down to the bottom, find the Rsyslog tile, and click "Use Integration":
+
+<ImgUseSyslogIntegration />
+
+Select the endpoint you just created from the dropdown and click "Enable". Then
+dismiss the integrations screen.
+
+After a few minutes, the Service Integrations item should refresh automatically and
+show your integration as active:
+
+<ImgSyslogIntegrationActive />
+
+<PublicOnly>
```

Comment:
Comment on the in-app experience for the spinner here - the notion of "once you've run the test command" is a bit odd, maybe we should special case that for platforms where we're streaming instead of downloading:

<img width="1095" alt="Screen Shot 2022-07-11 at 7 13 34 PM" src="https://user-images.githubusercontent.com/7227/178393906-fe1e424f-36d3-4caf-90b2-91a385440b48.png">

Reply:
Good point, but also a preexisting issue for other platforms; let's handle it separately.